script/stop-update.lua

```diff
@@ -77,22 +78,17 @@ function UpdateStop(stopID)
   stop.providePriority = 0
   stop.lockedSlots = 0
 
-  -- reject any stop not in name list
+  -- readd any stop not in name list
   if not global.TrainStopNames[stop.entity.backer_name] then

```

Comment:
When a stop is not in global.TrainStopNames something went horribly wrong.
Adding such a stop to normal processing cycle could introduce problems I can't even think of.